### PR TITLE
Gpg key multiple addresses

### PR DESCRIPTION
**Pull request by esroyo**
This pull request solves issue #672 by:
* Searching in all the addresses of a given key (not just the primary) when encrypting/decrypting a message.
* Showing always all the addresses of a given key in the dialogs (message composition, message decryption and OpenPGP settings).

This does not include:
* Creation of multi-address OpenPGP keys.

Moreover this request includes:
* Add sender key to the encryption keys so the email can be later read by the sender itself on the "Sent" folder (I believe  this is how email clients usually do work).
* Enable encrypt/decrypt buttons only if user inputs required fields (for instance, password).

### CODE REVIEW

**Comment by RainLoop** [comment]
Wow!

File: rainloop/v/0.0.0/app/templates/Views/User/PopupsMessageOpenPgp.html [hunk 1]
```diff
@@ -31,13 +37,13 @@
...
 							<input type="password" class="inputPassword input-xlarge" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"
-								data-bind="value: password, hasfocus: password.focus, onEnter: doCommand" />
+								data-bind="textInput: password, hasfocus: password.focus, onEnter: doCommand" />
 						</div>
 					</div>
 				</div>
 			</div>
 			<div class="modal-footer">
-				<button class="btn buttonDo" data-bind="command: doCommand, hasfocus: buttonFocus">
+				<button class="btn buttonDo" data-bind="command: doCommand, hasfocus: buttonFocus, enable: password().length > 0">
```
Comment (RainLoop) [inline rainloop/v/0.0.0/app/templates/Views/User/PopupsMessageOpenPgp.html]: Ok, what if private key does not require password?

Comment (esroyo) [inline rainloop/v/0.0.0/app/templates/Views/User/PopupsMessageOpenPgp.html]: You are right. I'll revert this and add another fix ASAP. Thanks!

Comment (RainLoop) [inline rainloop/v/0.0.0/app/templates/Views/User/PopupsMessageOpenPgp.html]: Please, wait for a while, Today will be a big commit with a lot of small fixes.

**Pull request merged**